python/lsst/pex/config/registry.py

```diff
@@ -263,15 +263,15 @@ def apply(self, *args, **kw):
         """
         if self.active is None:
             msg = "No selection has been made.  Options: %s" % \
-                (" ".join(list(self._field.typemap.registry.keys())))
+                (" ".join(list(self.ypes.registry.keys())))
```

Comment:
typo: `self.ypes`

There's also a set of do-nothing parentheses here, and you don't need the `list` call, either.  All of that's pre-existing, but might as well fix it since you need to touch the line again.